src/Validation/Rules/KnownTypeNamesRule.hack

```diff
@@ -0,0 +1,14 @@
+namespace Slack\GraphQL\Validation;
+
+final class KnownTypeNamesRule extends ValidationRule {
+    <<__Override>>
+    public function enter(\Graphpinator\Parser\Node $node): void {
+        if ($node is \Graphpinator\Parser\TypeRef\NamedTypeRef) {
+            $schema = $this->context->getSchema();
+            $type = (new $schema())->getIntrospectionType($node->getName());
```

Comment:
What do y'all think about always making the schema be an object (i.e., non-static)? It would be nice to not have to juggle between static and non-static methods.

Reply:
Seems reasonable, maybe a singleton like all the type classes?

`SchemaClass::get()->whatever()`

Reply:
How would `get()` know to return our generated schema? Unless `SchemaClass` is itself generated — but then how would this method know which `SchemaClass` to reference? We can't control the namespace under which the generated code will live.

Unless I'm missing something, I think it's easier to just pass the schema down from the resolver.